cpp/src/IceStorm/Scanner.l

```diff
@@ -217,6 +217,8 @@ keyword [[:alpha:]]*
 
 %%
 
+// NOLINTEND
```

Comment:
You didn't put it at the very end because you expect the code below to be lint-free?

Reply:
Yes, all the code under this line is _our_ code, copied verbatim into the generated file.

I don't expect it to be lint free (but maybe it is). But since it's our code, these are lints that we should be fixing, and not just ignoring!